.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
+<!-- If you are new to the Delta-V repository, please read the [Contributing Guidelines](https://github.com/DeltaV-Station/Delta-v/blob/master/CONTRIBUTING.md) -->
```

Comment:
We could just use the wiki if you want to, we have the means

Reply:
i can install SyntaxHighlight or something if you want me to

Reply:
pro
![image](https://github.com/user-attachments/assets/ea1afc05-d91e-46d5-b743-82f42c631c54)

Reply:
still good to have here as well so you can read it offline or if the wiki dies (not that it ever would :trollface:)